README.md

```
@@ -18,7 +18,7 @@ To see how to get started, check out the [StudioCMS Docs](./packages/studiocms/R
 ## Sponsors
 
 <a href="https://tur.so/studiocms" rel="sponsored" target="_blank"><img src="https://turso.tech/logokit/turso-logo-illustrated.svg" width="400px" /></a>
-<a href="https://www.skip2.net/?utm_source=studiocms" rel="sponsored" target="_blank"><img src="./assets/skip2-wordmark-red.svg" width="400px" /></a>
+<a href="https://www.skip2.net/?utm_source=studiocms" rel="sponsored" target="_blank"><img src="https://www.skip2.net/images/logo.svg" width="400px" /></a>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)